app/config.py

```diff
@@ -44,7 +44,6 @@ class QueueNames(object):
     RATE_LIMIT_RETRY = 'rate-limit-retry-tasks'
     NOTIFY = 'notify-internal-tasks'
     PROCESS_FTP = 'process-ftp-tasks'
-    CREATE_LETTERS_PDF = 'create-letters-pdf-tasks'
```

Comment:
Please remove the [commented code](https://github.com/department-of-veterans-affairs/notification-api/blob/5504543d6c9b99cbe706ea7524dbd39cd59f1f90/tests/app/test_config.py#L73) in tests/app/test_config.py that references this one. Also check l75 in that file to see if it can be commented out, please.

Reply:
Ok, I have removed the other commented out lines